### ISSUE TITLE
Logs Spam with database error

### ISSUE THREAD

**Issue by idkw** [Pterodactyl/Panel#428]
Hello,

So far, the panel works great and I thank you for this amazing work.
Right now, I have a strange issue. The framework is throwing an exception every second which add up to ~150MB of logs per day in .../storage/logs/ :

The same error is repeated over and over every second :

```
[2017-05-05 00:00:01] production.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function beginTransaction() on null in /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Database/Connection.php:605
Stack trace:
#0 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Queue/DatabaseQueue.php(175): Illuminate\Database\Connection->beginTransaction()
#1 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(173): Illuminate\Queue\DatabaseQueue->pop('high')
#2 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(143): Illuminate\Queue\Worker->getNextJob(Object(Illuminate\Queue\DatabaseQueue), 'high')
#3 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(76): Illuminate\Queue\Worker->runNextJob('database', 'high,standard,l...', Object(Illuminate\Queue\WorkerOptions))
#4 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(100): Illuminate\Queue\Worker->daemon('database', 'high,standard,l...', Object(Illuminate\Queue\WorkerOptions))
#5 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(83): Illuminate\Queue\Console\WorkCommand->runWorker('database', 'high,standard,l...')
#6 [internal function]: Illuminate\Queue\Console\WorkCommand->fire()
#7 /var/www/html/panel/bootstrap/cache/compiled.php(1272): call_user_func_array(Array, Array)
#8 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Console/Command.php(169): Illuminate\Container\Container->call(Array)
#9 /var/www/html/panel/vendor/symfony/console/Command/Command.php(261): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Console/Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /var/www/html/panel/vendor/symfony/console/Application.php(817): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /var/www/html/panel/vendor/symfony/console/Application.php(185): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Queue\Console\WorkCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 /var/www/html/panel/vendor/symfony/console/Application.php(116): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 /var/www/html/panel/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(121): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 /var/www/html/panel/artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 {main} 
```

I don't know what database it tries to connect to because the only database I have to store the servers parameters works properly because I'm able to run my servers.

Also, I have another issue that may be related. I tried to add a scheduled task on a server.
I've added a command task with the data `say lala` (also tried `/say lala`).
In the panel, it shows that the task was processed but in the console or in game, the command wasn't executed because nothing showed up in the chat.

Thank you

**Comment by parkervcp** [comment]
Hey @idkw You should join the discord to get help with things like this. But for simple answering  make sure to chown the files again to make sure they are owned correctly. It gets weird sometimes.

**Comment by DaneEveritt** [comment]
This looks like the Queue manager going off the rails. Should be all fixed in `v0.6.0`. Easiest fix right now is to just restart the queue worker, using `service supervisor restart`.
